runtime/src/iree/hal/local/elf/testdata/elementwise_mul.mlir

```diff
@@ -36,7 +36,7 @@ hal.executable.source public @ex {
   //
   // The ordinal is used to specify the entry point on command line tools and
   // must be unique across all entry points within the same executable.
-  hal.executable.entry_point public @elementwise_mul ordinal(0) layout(#executable_layout)
+  hal.executable.export public @elementwise_mul ordinal(0) layout(#executable_layout)
```

Comment:
a few "entry points" comments could say "exports" now

Reply:
I'll do a quick pass through - an export _is_ an entry point / entry points _are modeled by_ exports and many places talk about the concept of an entry point instead of specifically a `hal.executable.export` op.

Reply:
Gotcha. Both names make sense to me, but consistency is nice, and this sort of renaming tends to leave behind some outdated comments unless we're careful :P